starknet_py/tests/e2e/mock/contracts_v1/Scarb.toml

```diff
@@ -0,0 +1,12 @@
+[package]
```

Comment:
Should the Scarb.lock for the file be empty? of so, should it be included in the repo?

Reply:
IMO remove it, but no strong opinion